packages/coinstac-ui/app/render/components/consortia/consortia-list.jsx

```diff
@@ -195,8 +200,54 @@ class ConsortiaList extends Component {
     this.props.removeUserRole(user.id, 'consortia', consortiumId, 'member');
   }
 
-  startPipeline(consortiumId) {
+  startPipeline(consortiumId, activePipelineId) {
     return () => {
+      const { client } = this.props;
+      let isLocalPipeline = false;
+      const pipelineData = client.readQuery({ query: FETCH_ALL_PIPELINES_QUERY });
+      const pipeline = pipelineData.fetchAllPipelines
+        .find(pipe => pipe.id === activePipelineId);
+
+      for (let i = 0; i < pipeline.steps.length; i += 1) {
```

Comment:
I think it might make more sense if the last step isn't `remote`, as you could have several local preproc steps, and the final output is what matters for now

Reply:
This is backwards and is fixed in #380 to look for the existence of a `decentralized` step, which will flip the flag, `isRemotePipleine` to true and break.